.gitignore

```diff
@@ -8,6 +8,12 @@
 # Build & export artifacts
 build/
 
-# From Tiny Swords pack
+# macOS system files
+.AppleDouble
 .DS_Store
-.aseprite
```

Comment:
How is it that we had  `.aseprite` before but we managed to commit a lot of them? Like `scenes/game_elements/characters/enemies/guard/components/Storyvores.aseprite`

Reply:
Because this only ignores files whose name is `.aseprite`, not files whose names end with `.aseprite`. If this line had said `*.aseprite` then those files would have been ignored.